### ISSUE TITLE
[Feature Request] The Table Component Can't Sort By Multiple Properties

### ISSUE THREAD

**Issue by xiongkailing** [ElemeFE/element#14398]
### Existing Component
是

### Component Name
el-table

### Description
i have a table with multiple properties need to sort at the same time, but the el-table just one column can be sorted

**Comment by xiongkailing** [comment]
the 'sortable' property is 'custom'

**Comment by dhthomas76** [comment]
I agree with this feature request. We're porting most of our applications over to Vue.js from AngularJS, and the angular-ui grid (http://ui-grid.info/) we're currently using supports multi-column sorting natively. Having this feature supported by el-table natively would be a huge help to our users because without it they would lose functionality they currently have today.

**Comment by ped59430** [comment]
Do you have forecast on when the milestone 3.0 will be released, and if this feature will be part of it? Thanks

**Comment by ped59430** [comment]
#7689 might be related
